Sources/ViewController+Swinject.swift

```diff
@@ -14,12 +14,12 @@ private var uivcRegistrationNameKey: String = "UIViewController.swinjectRegistra
 private var uivcWasInjectedKey: String = "UIViewController.wasInjected"
 
 extension UIViewController: RegistrationNameAssociatable, InjectionVerifiable {
-    internal var swinjectRegistrationName: String? {
+    public var swinjectRegistrationName: String? {
```

Comment:
It's not sure we really need to make the properties public. We might revert it to internal later.